channels/es.m3u

```diff
@@ -323,28 +323,18 @@ https://cdn01.yowi.tv/lleida/playlist.m3u8
 http://streamer1.streamhost.org/salive/logosH/playlist.m3u8
 #EXTINF:-1 tvg-id="LogosTV.es" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://proyectoesperanza.es/wp-content/uploads/2017/02/logos-logo-big.png" group-title="",Logos TV (720p)
 https://streamer1.streamhost.org/salive/logosH/master.m3u8
-#EXTINF:-1 tvg-id="LogosTV.es" tvg-country="ES" tvg-language="" tvg-logo="" group-title="",Logos TV [Offline]
-http://68.235.60.118:1935/logoschannel.com/logostv.smil/playlist.m3u8
```

Comment:
Can be grabbed from here
https://logoschannel.com/live.aspx

Reply:
https://5aafcc5de91f1.streamlock.net/logoschannel.com/logostv.smil/playlist.m3u8 Arabic
https://5aafcc5de91f1.streamlock.net/logoschannel.com/logoseng/playlist.m3u8 English
https://5aafcc5de91f1.streamlock.net/logoschannel.com/logoshym/playlist.m3u8 Can't really tell, i'm confused lol.

Reply:
Added to `us.m3u`. These are two different channels:

https://www.lyngsat.com/tvchannels/us/Logos-TV.html
https://logostv.es/